src/content/deadlines/04_photos.mdx

```diff
@@ -1,8 +1,10 @@
 ---
-title: Photos
-subtitle: Refresh you memories!
-url: /photos
+title: 2024 Videos
```

Comment:
If you replace the photos with videos, I think there should still be a prominent place for photos on the homepage (footer is not prominent enough). 

Personally, I think it actually makes sense to replace photos with schedule. And then put schedule here on the secondary CTA here: https://github.com/EuroPython/website/blob/2c1254dac5ba79a46adf671f5c2372f6ccf37bd0/src/pages/index.astro#L56-L64 
Or you can put photos there, whichever you prefer.

Reply:
I moved the Photos down there for two reasons:
1. I wanted to have the schedule next to the recordings so visitors don't have to play connect the dots.
2. I was too lazy to create a new card picture for the recordings.